lib/zold/node/front.rb

```diff
@@ -267,8 +267,8 @@ class Front < Sinatra::Base
       JSON.pretty_generate(
         version: settings.version,
         score: score.to_h,
-        mtime: settings.remotes.mtime.utc.iso8601,
-        all: settings.remotes.all
+        all: settings.remotes.all,
+        mtime: settings.remotes.mtime.utc.iso8601
```

Comment:
@yegor256 This is the offending line that breaks the _test_all_scripts_ test. If you remove it the test is green, red otherwise.

However, the code works perfectly as I also tried to call the _remote_ endpoint from the same test, and the generated JSON is what is expected. Any comment on this?

Reply:
@yegor256 It is the _redeploy_on_upgrade_ that is failing, I cannot understand why. Could you explain how it works?

Reply:
@yegor256 This has been solved in mast commit, thanks anyway